src/order/bounded_order.lean

```diff
@@ -1737,6 +1762,61 @@ instance : complemented_lattice αᵒᵈ :=
 
 end complemented_lattice
 
+-- TODO: Define as a sublattice?
+/-- The sublattice of complemented elements. -/
+@[derive partial_order]
+def complementeds (α : Type*) [lattice α] [bounded_order α] : Type* := {a : α // is_complemented a}
+
+namespace complementeds
+section lattice
+variables [lattice α] [bounded_order α] {a b : complementeds α}
+
+instance has_coe_t : has_coe_t (complementeds α) α := ⟨subtype.val⟩
+
```

Comment:
More ways to go between the subtype and its property. I'm not convinced about the name for `complementeds.is_complemented` because it is quite long. It corresponds to `subtype.prop` so maybe call this `prop` as well?
```suggestion

protected lemma is_complemented (a : complementeds α) : is_complemented a := a.2

lemma exists_iff {a : α} : ∃ a' : complementeds α, a = a' ↔ is_complemented a := sorry

```

Reply:
In fact, there's no need to add `complementeds.is_complemented` since `a.prop : is_complemented ↑a`.